great_expectations/render/renderer/site_builder.py

```diff
@@ -100,6 +103,13 @@ def __init__(self,
                  datasource_whitelist=None
     ):
         self.data_context = data_context
+        self.store_backend = store_backend
+        
+        # set custom_styles_directory if present
+        custom_styles_directory = None
+        plugins_directory = data_context.plugins_directory
+        if os.path.isdir(os.path.join(plugins_directory, "custom_data_docs", "styles")):
+            custom_styles_directory = os.path.join(plugins_directory, "custom_data_docs/styles")
```

Comment:
Moved definition of custom_styles_directory from View to SiteBuilder, which is passed into View __init__